src/theme/defaultTheme/button.ts

```diff
@@ -18,18 +18,18 @@ export const button = {
     xl:
       "lib:h-12 lib:w-auto lib:min-w-12 lib:px-4 lib:text-base lib:font-medium lib:rounded-lg lib:shadow-sm",
   },
+  group: "lib:focus:z-1 lib:-mr-px lib:shadow-none",
   prefix: {
-    xs: "lib:inherit lib:mr-1.5",
-    sm: "lib:inherit lib:mr-2",
-    lg: "lib:inherit lib:mr-2",
-    xl: "lib:inherit lib:mr-2",
+    xs: "inherit mr-1.5",
+    sm: "inherit mr-2",
+    lg: "inherit mr-2",
+    xl: "inherit mr-2",
   },
   suffix: {
-    xs: "lib:inherit lib:ml-1.5",
-    sm: "lib:inherit lib:ml-2",
-    lg: "lib:inherit lib:ml-2",
-    xl: "lib:inherit lib:ml-2",
```

Comment:
Since, we are not provide and option to override the classnames for the prefix & suffix.
I thought we can just use the normal utility classes.

Reply:
Yup should just be fine